R/combined_seasonal_output.R

```diff
@@ -93,7 +93,7 @@ combined_seasonal_output <- function(
                                           disease_threshold = disease_threshold, n_peak = n_peak,
                                           family = family_quant, only_current_season = only_current_season, ...)
 
-  onset_output <- seasonal_onset(tsd = tsd, k = k, level = level, disease_threshold = disease_threshold,
+  onset_output <- seasonal_onset(tsd = tsd, k = k, level = level, disease_threshold = disease_threshold,       # nolint: lintrobject_usage_linter
```

Comment:
With the default `.lintr` configuration it is important to end with a period `.`
```suggestion
  onset_output <- seasonal_onset(tsd = tsd, k = k, level = level, disease_threshold = disease_threshold,       # nolint: object_usage_linter.
```

Reply:
Ohterwise, all linters are disabled